src/style.css

```diff
@@ -1 +1,42 @@
 /* add styles here */
+
+.page__body {
+  margin: 0;
+  gap: 0;
+  opacity: 0;
```

Comment:
The 'opacity' property is set to 0, which makes the whole body of the page invisible. If this is not the intended behavior, you should remove or adjust this property. This is a bug.